actors/builtin/miner/monies_test.go

```diff
@@ -73,4 +73,38 @@ func TestPledgePenaltyForTermination(t *testing.T) {
 				bigInitialPledgeFactor))
 		assert.Equal(t, expectedFee, fee)
 	})
+
+	t.Run("fee for replacement = fee for original sector when power, BR are unchanged", func(t *testing.T) {
+		// initialPledge equal to undeclaredPenalty guarantees expected reward is greater
+		initialPledge := undeclaredPenalty
+		dayReward := big.Div(initialPledge, bigInitialPledgeFactor)
+		twentyDayReward := big.Mul(dayReward, bigInitialPledgeFactor)
+		sectorAgeInDays := int64(20)
+		sectorAge := abi.ChainEpoch(sectorAgeInDays * builtin.EpochsInDay)
+		replacementAge := abi.ChainEpoch(2 * builtin.EpochsInDay)
+
+		// use low power, so we don't test SP=SP
+		power := big.NewInt(1)
+
+		// fee for old sector if had terminated when it was replaced
+		unreplacedFee := miner.PledgePenaltyForTermination(
+			dayReward,
+			big.Zero(),
+			twentyDayReward,
+			sectorAge,
+			0,
+			rewardEstimate, powerEstimate, power)
+
+		// actual fee including replacement parameters
+		actualFee := miner.PledgePenaltyForTermination(
+			dayReward,
+			dayReward,
+			twentyDayReward,
+			replacementAge,
+			sectorAge-replacementAge,
+			rewardEstimate, powerEstimate, power)
+
+		// newFee + oldFee double counts the twentyDay reward so subtract it from expectation
+		assert.Equal(t, unreplacedFee, actualFee)
+	})
```

Comment:
I would like to see another test here exercising a more canonical case where the replacement br and 20br are different. Perhaps even spelling out the actual expected result (duplicating the calculation logic for testing).